### PR TITLE
NISRA routing change

### PR DESCRIPTION
**Pull request by petechd**
### What is the context of this PR?

This adds small routing change based on fixed date as described on this [Trello card](https://trello.com/c/ScuwSC4d).

### How to review

Check if changes are working as expected in Northern Ireland census schemas.

#### Northern Ireland

- [Household](https://test-launcher.gcp.dev.eq.ons.digital/quick-launch?language_code=en&survey=CENSUS&form_type=I&region_code=GB-NIR&url=https://storage.googleapis.com/eq-questionnaire-schemas-artifacts/nisra-routing-change/schemas/en/census_household_gb_nir.json)

- [Individual](https://test-launcher.gcp.dev.eq.ons.digital/quick-launch?language_code=en&survey=CENSUS&form_type=I&region_code=GB-NIR&url=https://storage.googleapis.com/eq-questionnaire-schemas-artifacts/nisra-routing-change/schemas/en/census_individual_gb_nir.json)

### CODE REVIEW

**Comment by petechd** [comment]
[[A] NISRA routing change (XS)](https://trello.com/c/ScuwSC4d/3949-a-nisra-routing-change-xs)

File: source/jsonnet/northern-ireland/lib/common_rules.libsonnet [hunk 1]
```diff
@@ -69,6 +69,16 @@
       },
     },
   },
+  nisraOver5: {
+    id: 'date-of-birth-answer',
+    condition: 'less than or equal to',
+    date_comparison: {
+      value: '2020-07-01',
```
Comment (LJBabbage) [inline source/jsonnet/northern-ireland/lib/common_rules.libsonnet:76]: The logic works, but the date on the card is 1 July 2021, so one is wrong

Comment (petechd) [reply source/jsonnet/northern-ireland/lib/common_rules.libsonnet:76]: I used existing nisraUnder4 which might have a wrong date attached, I need to double check.

Comment (petechd) [reply source/jsonnet/northern-ireland/lib/common_rules.libsonnet]: Ok, so it was supposed to be 2021, great catch, many thanks!

File: source/jsonnet/northern-ireland/lib/common_rules.libsonnet [hunk 2]
```diff
@@ -63,12 +63,22 @@
...
     condition: 'greater than',
     date_comparison: {
-      value: '2020-07-01',
+      value: '2021-07-01',
       offset_by: {
         years: -4,
       },
     },
   },
+  nisraOver5: {
```
Comment (ajmaddaford) [inline source/jsonnet/northern-ireland/lib/common_rules.libsonnet:72]: This name confused me as this is a NISRA specific rules file and we already have an over5 rule. I asked Donna what the significance of it was, she said:

> Anyone aged 4 on 1st July of that year should go to school. If you are aged 4 on 2nd July you have to wait until the next year so shouldn't see any questions on schools.

I think we should rename:
`nisraUnder4` -> `schoolYearUnder4`
`nisraOver5` -> `schoolYearOver5`

Comment (ajmaddaford) [reply source/jsonnet/northern-ireland/lib/common_rules.libsonnet:72]: I've just noticed it's less than or equal to, so it should be something like `schoolYear5OrOver`.

Comment (petechd) [reply source/jsonnet/northern-ireland/lib/common_rules.libsonnet]: Renamed according to your suggestions, thanks👍

**Pull request merged**